stdlib/inspect.pyi

```diff
@@ -161,12 +161,20 @@ TPFLAGS_IS_ABSTRACT: Literal[1048576]
 
 modulesbyfile: dict[str, Any]
 
+_GetMembersPredicateTypeGuard: TypeAlias = Callable[[_T], TypeGuard[_T]]
```

Comment:
Shouldn't this be:

```suggestion
_GetMembersPredicateTypeGuard: TypeAlias = Callable[[Any], TypeGuard[_T]]
```

I don't know of any `TypeGuard` functions that only accept objects that are already known to be of the type the `TypeGuard` function is verifying. If a type checker complains, you could try writing it as a generic callback protocol instead:

```python
class _GetMembersPredicateTypeGuard(Protocol[_T]):
    def __call__(self, __obj: Any) -> TypeGuard[_T]: ...
```

It might actually be more readable as a callback protocol, in this case.